.github/workflows/release.yml

```diff
@@ -37,21 +37,12 @@ jobs:
         run: |
           echo "::notice::Release v${{ github.event.inputs.version }} from branch ${{ github.ref_name }}"
 
-      - name: Install semver CLI
-        run: |
-          local_bin="${HOME}/.local/bin"
-          mkdir -p "${local_bin}"
-          curl -L --output "${local_bin}/semver" \
-            https://raw.githubusercontent.com/fsaintjacques/semver-tool/3.3.0/src/semver
-          chmod +x "${local_bin}/semver"
-          echo "${local_bin}" >> $GITHUB_PATH
-
-      - name: Validate release version
-        run: |
-          if [ "$(semver validate ${{ github.event.inputs.version }})" == "invalid" ]; then
-            echo "::error::Version ${{ github.event.inputs.version }} is invalid"
-            exit 1
-          fi
+      - name: Parse the version
+        id: version
+        uses: madhead/semver-utils@v2.4.0
```

Comment:
Circling back on this a bit, I'm confused by this tag. The https://github.com/marketplace/actions/semver-utils URL shows the latest version being 1.0.0. And the Action tag doesn't seem to have any relationship to the semver utils version (for which we were using 3.3.0).

Reply:
The action version indeed doesn't have any relation to the semver library version. That's intentional.

The repository does actually contain a `v2.4.0` tag: https://github.com/madhead/semver-utils/tree/v2.4.0. There is also a `v2.4.0` release in the releases: https://github.com/madhead/semver-utils/releases. I don't know why the Markeplace uses 1.0.0 as the version and I don't see any knobs to refresh it there.